olive/passes/onnx/conversion.py

```diff
@@ -55,26 +66,28 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
 
     def _run_for_config(
         self, model: PyTorchModel, data_root: str, config: Dict[str, Any], output_model_path: str
-    ) -> Union[ONNXModel, CompositeOnnxModel]:
-        return self._convert_model_on_device(model, data_root, config, output_model_path, "cpu")
```

Comment:
the original model conversion will always happen in CPU side. @trajepl should have some comments. I am not sure this behavior change has any impact.

Reply:
so, we don't need convert on GPU like https://github.com/microsoft/Olive/pull/478 anymore?

Reply:
We don't have an example where it's at the moment and as @devang-ml suggested it should be implemented differently. So, instead of fixing existing unused code, I chose to remove it.